src/site/layouts/config.js

```diff
@@ -0,0 +1,64 @@
+const { ENTITY_BUNDLES } = require('../constants/content-modeling');
+
+const directoryStructure = {
+  'va-medical-centers': [
+    ENTITY_BUNDLES.BIOS_PAGE,
+    ENTITY_BUNDLES.EVENT,
+    ENTITY_BUNDLES.EVENTS_PAGE,
+    ENTITY_BUNDLES.EVENT_LISTING,
+    ENTITY_BUNDLES.HEALTH_CARE_FACILITY_STATUS,
+    ENTITY_BUNDLES.HEALTH_CARE_LOCAL_FACILITY,
+    ENTITY_BUNDLES.HEALTH_CARE_LOCAL_HEALTH_SERVICE,
+    ENTITY_BUNDLES.HEALTH_CARE_REGION_DETAIL_PAGE,
+    ENTITY_BUNDLES.HEALTH_CARE_REGION_LOCATIONS_PAGE,
+    ENTITY_BUNDLES.HEALTH_CARE_REGION_PAGE,
+    ENTITY_BUNDLES.HEALTH_SERVICES_LISTING,
+    ENTITY_BUNDLES.LEADERSHIP_LISTING,
+    ENTITY_BUNDLES.LOCATIONS_LISTING,
+    ENTITY_BUNDLES.NEWS_STORIES_PAGE,
+    ENTITY_BUNDLES.NEWS_STORY,
+    ENTITY_BUNDLES.OFFICE,
+    ENTITY_BUNDLES.PERSON_PROFILE,
+    ENTITY_BUNDLES.PRESS_RELEASE,
+    ENTITY_BUNDLES.PRESS_RELEASES_LISTING,
+    ENTITY_BUNDLES.PUBLICATION_LISTING,
+    ENTITY_BUNDLES.REGIONAL_HEALTH_CARE_SERVICE_DES,
+    ENTITY_BUNDLES.STORY_LISTING,
+    ENTITY_BUNDLES.SUPPORT_SERVICE,
+    ENTITY_BUNDLES.VAMC_OPERATING_STATUS_AND_ALERTS,
+  ],
+  'resources-and-support': [
+    ENTITY_BUNDLES.BASIC_LANDING_PAGE,
+    ENTITY_BUNDLES.CHECKLIST,
+    ENTITY_BUNDLES.FAQ_MULTIPLE_Q_A,
+    ENTITY_BUNDLES.MEDIA_LIST_IMAGES,
+    ENTITY_BUNDLES.MEDIA_LIST_VIDEOS,
+    ENTITY_BUNDLES.Q_A,
+    ENTITY_BUNDLES.STEP_BY_STEP,
+    ENTITY_BUNDLES.SUPPORT_RESOURCES_ARTICLE_LISTING,
+    ENTITY_BUNDLES.SUPPORT_RESOURCES_DETAIL_PAGE,
+  ],
+  'benefit-hubs': [
+    ENTITY_BUNDLES.HOME,
+    ENTITY_BUNDLES.LANDING_PAGE,
+    ENTITY_BUNDLES.PAGE,
+    ENTITY_BUNDLES.VA_FORM,
+  ],
+  other: [
+    ENTITY_BUNDLES.FULL_WIDTH_BANNER_ALERT,
+    ENTITY_BUNDLES.OUTREACH_ASSET,
+  ],
+};
+
+const layoutFilesByEntityBundle = new Map();
+
+for (const [directory, entityBundles] of Object.entries(directoryStructure)) {
+  for (const entityBundle of entityBundles) {
+    layoutFilesByEntityBundle.set(
+      entityBundle,
+      `${directory}/${entityBundle}.drupal.liquid`,
+    );
+  }
+}
+
+module.exports = layoutFilesByEntityBundle;
```

Comment:
My main concern is this file adding complexity. I hope that by being located at `site/layouts/config.js`, it's clear to an engineer what it does. A common mystery to a new engineer is "I have to create a new layout - how do I do that?"  I'm hoping that instead of adding complexity, this might make it clearer

Reply:
It might be nice to add a comment at the top of this file indicating its intention/purpose. Also I'm wondering if this increases the probability of duplicate templates being created, since hypothetically (for example) a step by step page template doesn't have to belong exclusively to R&S. Not saying we shouldn't do this, just want to bring it up as a point of discussion

Reply:
Completely agree on both points...

1. A comment -> definitely, yeah, could be a good chance to add documentation there similar to how .NET projects added giant comments into their config files IIRC. Will do!
1. Duplicate templates -> I think we would hope that we would maintain the project well enough to move shared templates out of certain directories to indicate that they are not confined to a certain section anymore. But yeah, what is the likelihood that would happen...It's basically a tradeoff then - organizing now, but risking the organization not being kept up